docs/modules/wan/pages/defining-wan-replication.adoc

```diff
@@ -317,9 +317,9 @@ hazelcast:
 See the following for the configurations of WAN replications in other cloud infrastructures
 that are supported by Discovery SPI:
 
-* https://github.com/hazelcast/hazelcast-azure/blob/master/README.md#configuration-for-wan-replication-target-cluster-discovery[Microsoft Azure^]
-* https://github.com/hazelcast/hazelcast-gcp/blob/master/README.md#configuration[Google Cloud Platform (GCP)^]
-* https://github.com/hazelcast/hazelcast-kubernetes#configuration[Kubernetes^]
```

Comment:
We also need to change the links on the `migration-guides.adoc` page.

Reply:
@Serdaro it looks like the `migration-guides.adoc` page should be removed from 5.0. It only discusses IMDG and upgrading to 4.x versions.

Reply:
That page has been removed from ROOT, if that's what you mean. It was also causing warnings (unterminated listing blocks).